taxi/views.py

```diff
@@ -85,6 +125,24 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
     model = Driver
     paginate_by = 5
 
+    def get_queryset(self):
```

Comment:
model and queryset are interchangeable so if you overriden get_queryset, which returns queryset for view, there's no need in model attribute, you can remove it.